### DISCUSSION TITLE
Please, add "LAST-MODIFIED" to the event properties

### DISCUSSION THREAD

**Discussion by jhernandis** [spatie/icalendar-generator#140] [Ideas]
I miss the LAST-MODIFIED property for the events items. Can you add it? Thanks.

**Comment by rubenvanassche** [comment]
You're always welcome to send in a PR adding this!

**Comment by jhernandis** [comment]
Great! I'll try it.

**Comment by jhernandis** [comment]
Meanwhile, I've added this code to our ICS generator:
`$icsEvent->appendProperty(DateTimeProperty::create('LAST-MODIFIED', new DateTimeValue(new DateTime($ev->event_updated, $tz))));`
